planning/behavior_path_planner/src/scene_module/avoidance/avoidance_utils.cpp

```diff
@@ -192,4 +192,24 @@ void setStartData(
   ap.start_longitudinal = start_dist;
 }
 
+std::string getUuidStr(const ObjectData & obj)
+{
+  std::stringstream hex_value;
+  for (std::size_t idx = 0; idx < 4; ++idx) {
```

Comment:
as the number of uuid characters to display, I don't really sure whether to display all 32 characters or only a portion of it.

As personally I think at this stage we don't really need to process all of them, so I only choose 8 characters ( note to myself): each byte consist of 2 hexadecimal characters, therefore 2x4 byte = 8 characters)

Reply:
Only for the visualization, 4 characters are enough i think. It is the same number as used in the perception debug visualization.

Reply:
In the debug topic, you can use the full characters.

Reply:
Alright. The current commit is updated to display all characters. 👍🏼